framework/meson.build

```diff
@@ -66,7 +66,7 @@ framework_config.set10('SANDSTONE_GA_DEV', get_option('flavor') == 'ga-dev')
 framework_config.set10('SANDSTONE_NO_LOGGING', get_option('flavor') == 'ga')
 framework_config.set10('SANDSTONE_RESTRICTED_CMDLINE', get_option('cmdline') == 'restricted')
 
-framework_config.set10('SANDSTONE_DEBUG', get_option('b_ndebug') == 'false')
+framework_config.set10('SANDSTONE_DEBUG', get_option('buildtype') != 'release')
```

Comment:
Actually, I'd like you **not** to set `SANDSTONE_DEBUG` at all. It isn't used any where, but `NDEBUG` and `DEBUG` are. Meson appears to automatically add `-DNDEBUG` to release builds, but #93 removed setting `DEBUG` if not. So we need to bring back this section of code:
```c
// Make sure only one of DEBUG / NDEBUG is defined
#if !defined(DEBUG) && !defined(NDEBUG)
#  define DEBUG 1
#endif
#if defined(DEBUG) && defined(NDEBUG)
#  undef NDEBUG
#elif defined(NDEBUG)
#  define DEBUG 0
#endif
#if DEBUG
#  define SANDSTONE_DEBUG       1
#else
#  define SANDSTONE_DEBUG       0
#endif
```

Reply:
Will do.